packages/accounts/src/msca/account/multiOwnerAccount.ts

```diff
@@ -90,27 +131,44 @@ export async function createMultiOwnerModularAccount({
     ]);
   };
 
-  accountAddress = await getAccountAddress({
+  const _accountAddress = await getAccountAddress({
     client,
-    entryPointAddress: entryPoint.address,
-    accountAddress: accountAddress,
-    getAccountInitCode,
-  });
-
-  const baseAccount = await toSmartContractAccount({
-    transport,
-    chain,
-    entryPoint,
+    entryPoint: _entryPoint,
     accountAddress,
-    source: `MultiOwnerModularAccount`,
     getAccountInitCode,
-    ...standardExecutor,
-    ...multiOwnerMessageSigner(client, accountAddress, () => signer),
   });
 
+  const baseAccount =
```

Comment:
same comment here as with light account, this doesn't support 0.7 so we shouldn't even allow devs to do this